pipelines/rj_smas__disparo_pic/utils/tasks.py

```
@@ -0,0 +1,207 @@
+
+    # Create and return authenticated API handler
+    api = ApiHandler(base_url=url, username=username, password=password, login_route=login_route)
+
+    return api
+
+
+@task
+def safe_export_df_to_parquet(
+    dfr: pd.DataFrame,
+    output_path: str = "./",
+    wait=None,  # pylint: disable=unused-argument
```

Ruff (0.14.1)

Unused function argument: `wait`

(ARG001)